haiku/_src/module.py

```diff
@@ -124,7 +124,7 @@ def __new__(  # pylint: disable=bad-classmethod-argument
 
     return cls
 
-  def __call__(cls: Type[T], *args, **kwargs) -> T:  # pylint: disable=no-self-argument
+  def __call__(cls: 'ModuleMetaclass', *args, **kwargs) -> Any:  # pylint: disable=no-self-argument
```

Comment:
Just a small nit: can you change to `"` to match the quote style used in this file?

Reply:
Done